doc/src/modules/physics/control/control.rst

```diff
@@ -4,17 +4,19 @@ Control
 
 Currently, :mod:`sympy.physics.control` is able to deal with LTI
 (Linear, time-invariant) systems. The ``TransferFunction`` class is used to
-represent Continuous-time Transfer functions in the Laplace domain; where Transfer
-functions are input to output representations of dynamic systems. The additive
-property is used for transfer functions in the ``Parallel`` class, and the
-multiplicative property is used for transfer functions in the ``Series`` class.
+represent Continuous-time Transfer functions in the Laplace domain, while
+``DTTransferFunction`` represent Discrete-time Transfer functions in the
```

Comment:
I think we should name this "DiscreteTranferFunction", i.e. use full words instead of single letters. I'm open for other options, but it is customary in Python to be descriptive with names.